### ISSUE TITLE
[Accessibility] - keyboard navigation on the player controls interferes with play/pause

### ISSUE THREAD

**Issue by ziegenberg** [openplayerjs/openplayerjs#261]
**Describe the bug**
In Firefox, when navigating to the player controls via `Tab` key (e.g. to focus the mute button) and pressing `Space` to toggle mute, the player automatically switches state from playing to pausing and vice versa. This also applies to the play button.

**To Reproduce**
1. Create a player
2. Navigate to the player controls via `Tab` key to focus the mute button
3. Press `Space` to toggle mute
4. The mute state is toggled and the player switches playing state and starts/stops playing the video

**Expected behavior**
Only the mute state is toggled.

**Desktop):**
 - OS: Fedora 34 / Windows 10
 - Browser: Firefox
 - Version: 90

**Additional context**
Toggling the mute button in Chrome with keyboard navigation does not work at all.

**Comment by rafa8626** [comment]
@ziegenberg please download the master branch and let me know if the new changes with keyboard shortcuts fixed this as well

**Comment by ziegenberg** [comment]
I can still confirm this issue with Firefox. It catches all `Space` key inputs either when the video element has focus or when specific controls have the focus and always toggles play/pause. I'll check with Chrome tomorrow.

**Comment by rafa8626** [comment]
@ziegenberg please download the master branch and update your URL with this new round of fixes. I tested it myself in Chrome and FF and it worked as expected. Let me know if it worked for you as well

**Comment by ziegenberg** [comment]
I tested on Fedora 34 with Chrome: it works now as expected.
I tested on Fedora 24 with Firefox: it's still broken. Toggling mute (or play) also stops/starts the player

I will test on Windows later this evening and report back with my findings.

**Comment by rafa8626** [comment]
@ziegenberg I checked this with OSX in Chrome, FF and Safari and all worked as expected. I don’t have a way to spin a Linux instance currently. There’s one more thing I can do to mitigate this but after that it would be hard to know what’s the issue without the environment where you are experiencing this

**Comment by rafa8626** [comment]
@ziegenberg Please download the master branch and confirm this works as expected on your side. I added a stop progration of events that should help in your scenario
